### PR TITLE
Path template in metrics

### LINKED ISSUE #5: [FEATURE] Path template instead of actual path in metrics
Hi, there!

Thanks for a great middleware! I've been using it a while and now I want to show response time by url in grafana. It works good with regular paths, like `/users`, but not with templated paths like `/users/{id}` because in `/metrics` they appear as actual paths (`/users/1`, `/users/2`, etc...)

I've made a quick pull request https://github.com/perdy/starlette-prometheus/pull/6 for this. Let me know what you think of this idea and feel free to decline it if anything

### PR DESCRIPTION
**Pull request by unmade**
Resolve #5

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@ exclude = []
 
 [tool.poetry.dependencies]
 python = "^3.6"
-starlette = ">=0.12.0"
+starlette = ">=0.12.2"
```
Comment (unmade) [inline pyproject.toml:19]: prior 0.12.2 the `scope` attribute of Request was private

File: starlette_prometheus/middleware.py [hunk 2]
```diff
@@ -49,3 +50,11 @@ class PrometheusMiddleware(BaseHTTPMiddleware):
             REQUESTS_IN_PROGRESS.labels(method=method, path=path).dec()
 
         return response
+
+    @staticmethod
+    def get_path(request: Request):
```
Comment (perdy) [inline starlette_prometheus/middleware.py]: It could be nice to have the signature of the response as well as parameters: `def ... -> str`.

Comment (unmade) [reply starlette_prometheus/middleware.py]: Done

File: tests/test_end_to_end.py [hunk 3]
```diff
@@ -63,3 +67,11 @@ def test_view_exception(self, client):
...
         assert 'starlette_requests_in_progress{method="GET",path="/bar/"} 0.0' in metrics_text
         assert 'starlette_requests_in_progress{method="GET",path="/metrics/"} 1.0' in metrics_text
+
+    def test_path_substituion(self, client):
+        client.get("/foo/baz")
+
+        response = client.get("/metrics/")
+        metrics_text = response.content.decode()
+
+        assert 'starlette_requests_total{method="GET",path="/foo/{bar}/"} 1.0' in metrics_text
```
Comment (perdy) [inline tests/test_end_to_end.py]: Is the test updated to reflect `path_template` changes ?

Comment (unmade) [reply tests/test_end_to_end.py]: Oh... totally forgot. I updated it now. 
Sorry for not answering so long!

**Comment by dedsm** [comment]
I'd love to get this merged, just stumbled into this

**Review by perdy — APPROVED** [review]

**Pull request merged**